cwms-aaa-client/src/main/java/mil/army/usace/hec/cwms/aaa/client/CacKeyManagerUtil.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2022
+ * United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
+ * All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+
+package mil.army.usace.hec.cwms.aaa.client;
+
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.X509KeyManager;
+
+
+public final class CacKeyManagerUtil {
+    private static KeyManager instance;
+
+    private CacKeyManagerUtil() {
+        throw new AssertionError("Utility class");
+    }
+
+    private static KeyManager createKeyManager() throws CacCertificateException {
```

Comment:
I had this kind of logic bite me once, nothing wrong with it by default but it might not hurt to make sure the behavior can be overridden without having to "hack" the os.name var.

Reply:
yeah, I moved this out of the source code and just added the vanilla JRE keystore to the test class since that's all that is currently needed.